src/utilities/SimulationBaseClass.py

```diff
@@ -373,18 +373,17 @@ def CreateNewPythonProcess(self, procName, priority = -1):
         self.pyProcList.append(proc)
         return proc
 
-    def CreateNewTask(self, TaskName, TaskRate, InputDelay=0, FirstStart=0):
+    def CreateNewTask(self, TaskName, TaskRate, FirstStart=0):
```

Comment:
:small_red_triangle_down: I think that we should be setting up a deprecation warning and maintaining the API such that it doesn't break for users who explicitly provide the parameter. I'm happy to suggest a method for doing this.

Reply:
Done